README.md

```diff
@@ -45,7 +45,7 @@ Jellyfin Web is the frontend used for most of the clients available for end user
 ### Dependencies
 
 - [Node.js](https://nodejs.org/en/download)
-- [Yarn 1.22.5](https://classic.yarnpkg.com/en/docs/install)
+- npm (included in Node.js)
```

Comment:
Will any NPM version or is there a minimum requires version? (Like 7>=)

Reply:
We've had problems with "old" versions of npm on Vue, right @ferferga ?

Reply:
@nielsvanvielzen Good thing you're asking this and thank you very much for @ThibaultNocchi four pointing it out.

The main feature we use yarn for is for deterministic packages. That has been the case for ``npm`` for quite a few years. That's the *minimum* we need to keep our workflow as equal to yarn as possible.

NodeJS 14 and 12 includes ``npm 6``, while package-lock.json was introduced in ``npm 5``. While npm 7 is supposedly more performant than the 6 and 5 counterparts, they work, so it's frictionless for everybody, while still keeping the freedom for those that want the greatest and latest and upgrade to npm 7.

Now, this leads us to the Vue problem: In my system I don't have the LTS version included with Node, as I updated it for other packages (I don't remember which version I had, but I was already in the 7.x branch)

I discovered that running ``npm ci`` (equivalent to ``--frozen-lockfile`` argument) command missed the dependencies of some dependencies. That made CI fail. Upgrading to 7.6.0 fixed the problem, so I took the easy way of amending my PR by updating always to latest npm version, and that fixed all the problems. 

So, **TLDR, npm install has always been reliable, the problem has been with npm ci**. So "users" or us were never affected *directly*

I need to investigate more which versions are exactly affected with this bug if we want to stick with the bundled npm versions during our CIs, instead of following my approach of always using the latest available.

Reply:
I confirm that 6.14 (the included version in Node 14) works perfectly both in ``install`` and ``ci``, so no need for the ugly hack I used in Vue.